pkgs/top-level/python-packages.nix

```
@@ -995,6 +995,8 @@ in {
 
   pylev = callPackage ../development/python-modules/pylev { };
 
+  pylibftdi = callPackage ../development/python-modules/pylibftdi { };
```

use this area to pass the individual packages, instead of the entire pkgs namespace, makes it easier to override
```suggestion
  pylibftdi = callPackage ../development/python-modules/pylibftdi { 
    inherit (pkgs) libusb1;
  };
```